packages/harpguru-core/src/components/option-value/option-value.tsx

```diff
@@ -0,0 +1,59 @@
+import { useTimingTransition } from 'react-native-redash'
+import Animated, { Easing, interpolate } from 'react-native-reanimated'
+import { TapGestureHandler, State } from 'react-native-gesture-handler'
+import type { TapGestureHandlerStateChangeEvent } from 'react-native-gesture-handler'
+import { View, Text, TextStyle } from 'react-native'
+import React, { useState } from 'react'
+
+import { OptionIds } from '../../types'
+
+type OptionValueProps = {
+  readonly id: OptionIds | undefined
+  readonly isActive: boolean
+  readonly setFunction: (arg0: OptionIds) => void
+  readonly style: TextStyle
+}
+
+export const OptionValue = ({
+  id,
+  isActive,
+  setFunction,
+  style,
```

Comment:
Do we need to pass the style down? Might the component have enough information to determine it itself now?

Reply:
Done